jerry-core/ecma/base/ecma-globals.h

```diff
@@ -292,6 +292,11 @@ typedef enum
 #define ECMA_PROPERTY_TYPE_NOT_FOUND ECMA_PROPERTY_TYPE_DELETED
 
 /**
+ * Type of property not found and no more searching in the proto chain.
+ */
+#define ECMA_PROPERTY_TYPE_NOT_FOUND_AND_STOP ECMA_SPECIAL_PROPERTY_VALUE (ECMA_INTERNAL_PROPERTY_NOT_FOUND_AND_STOP)
```

Comment:
Could we use some other property value such ECMA_PROPERTY_TYPE_HASHMAP, which cannot be a valid return value?